tests/equilibria/test_constraint_functions.py

```diff
@@ -115,6 +164,13 @@ def test_current_midplane_constraint(self):
             )
             assert cmc.f_constraint(self.vector) == pytest.approx(c)
 
+    def test_coil_force_constraint_against_old_function(self):
+        np.testing.assert_allclose(
+            old_control_F(self.coilset, self.coilset),
+            self.coilset.control_F(self.coilset),
+            atol=4e-6,
```

Comment:
This is a bad test. The absolute values are smaller than the tolerance. I checked though and this passes with atol=0.0, and rtol=0.0